internal/storage/bucket_handle.go

```diff
@@ -26,7 +26,7 @@ import (
 	"net/http"
 
 	"cloud.google.com/go/storage"
-	"github.com/googlecloudplatform/gcsfuse/internal/gcloud/gcs"
+	"github.com/googlecloudplatform/gcsfuse/internal/storage/gcs"
```

Comment:
the files under storage/gcs can directly go under storage.

Reply:
As I mentioned in the description
```
The files which are under gcs directory, I kept them in [gcs directory](https://github.com/GoogleCloudPlatform/gcsfuse/blob/34c4fcf3d464b8a1a69bd4fd052b21b04f96a865/internal/storage/gcs/bucket.go) only as If I was putting it in to storage directory It was creating cycle between storage and storagutil. 
```

Reply:
Got it, i missed that. In that case, I would suggest to create `object` and 'bucket' packages under storage/. So the hierarchy would like:
internal/storage/
    object/
        object.go (copied from gcloud/gcs)
        ... (other object related files copied from gcloud/gcs and storage/ if any)
    bucket/
         object.go (copied from gcloud/gcs)
        ... (other bucket related files copied from gcloud/gcs and storage/ if any)    
    
    bucket_handle.go
    storage_handle.go
    requests.go 
    ...
    
CC' @vadlakondaswetha Does it look good to you ?

Reply:
There are lot's of cycle forming due to this, I did some formation let me konw if it's correct.

Reply:
Can you please give example ?

Is it possible to keep the following?
- storage/
  - object/
    - object.go
    - object_test.go
    - object_attrs.go
    - object_attrs_test.go
  - bucket/
    - bucket.go
    - debug_bucket.go
    - mock_bucket.go
  - requests/
    - request.go
  - storageutil/
    - Move everything from gcsutil to here.
  - errors.go
  - bucket_handle.go
  - bucket_handle_test.go
  - storage_handle.go
  - storage_handle_test.go
  - fake_storage_util.go
  - all other files

Reply:
Object and request are forming cycle. Both should be in same directory.

Reply:
In that case, object_attrs.go & object_attrs_test.go should be kept under storageutil/. Do you see any other issue with that ?